02-typescript/02-todo-list/src/main.ts

```diff
@@ -1,9 +1,12 @@
 const getDiv = document.getElementById("to-do");
-
+const crtUl = document.createElement("ul");
+const crtInput = document.createElement("input");
+const crtFilter = <HTMLInputElement>document.getElementById("filter-todo");
```

Comment:
Be careful with that: the `<...>` type assertion (see https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions) does not check for `null`. This is not wrong, but I would generally prefer explicitly checking for `null`, as in:

```ts
if(!(crtFilter instanceof HTMLInputElement)) {
  throw new Error("No input with id 'filter-todo' found!");
}
```